src/headerdef.rs

```diff
@@ -0,0 +1,43 @@
+#[derive(Debug, Display, Clone, PartialEq, Eq, EnumVariantNames)]
+#[strum(serialize_all = "kebab_case")]
+#[allow(dead_code)]
+pub enum HeaderDef {
+    MessageId,
+    Subject,
+    Date,
+    From_,
+    To,
+    Cc,
+    Disposition,
+    OriginalMessageId,
+    ListId,
+    References,
+    InReplyTo,
+    Precedence,
+    ChatVersion,
+    ChatGroupId,
+    ChatGroupName,
+    ChatGroupNameChanged,
+    ChatVerified,
+    ChatGroupImage,
+    ChatVoiceMessage,
+    ChatGroupMemberRemoved,
+    ChatGroupMemberAdded,
+    ChatContent,
+    ChatDuration,
+    ChatDispositionNotificationTo,
+    AutocryptSetupMessage,
+    SecureJoin,
+    SecureJoinGroup,
+    SecureJoinFingerprint,
+    SecureJoinInvitenumber,
+    SecureJoinAuth,
+    _TestHeader,
```

Comment:
how's this converted to kebab-case?  i'd have no idea without seeing an example (may  i suggest a test ;))